src/index.js

```diff
@@ -1,8 +1,9 @@
 import React, { Component } from 'react';
 import csjs from 'csjs';
+import scopedName from 'csjs/lib/scoped-name';
```

Comment:
I think for now, to bridge the bigger goal, it should be fine, at least it would save us from duplicating some code. Later we can move on.